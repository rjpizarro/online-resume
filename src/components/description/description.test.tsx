import React from 'react'
import renderer from 'react-test-renderer'

import Description from './index'

jest.mock('../avatar-image', () => 'AvatarImage')

describe("Description", () => {
  it("renders normally", () => {
    const tree = renderer
      .create(<Description />)
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})