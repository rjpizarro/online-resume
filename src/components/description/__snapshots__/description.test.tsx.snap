// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description renders normally 1`] = `
<div
  className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-8"
>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-lg-8"
  >
    <p
      className="MuiTypography-root makeStyles-description-1 MuiTypography-body1"
    >
      Although I studied a B.A in Social Communication, I have always been a techie. That led me, around 10 years ago, to start a blog. I never thought that I was about to find a career also.
    </p>
    <p
      className="MuiTypography-root makeStyles-description-1 MuiTypography-body1"
    >
      Early on I found that I was able to edit the page's HTML and CSS. Very soon, like an owl, I was up all night learning and editing every line of code from my site.
    </p>
    <p
      className="MuiTypography-root makeStyles-description-1 MuiTypography-body1"
    >
      The years went by and I had the chance to learn and work with other tools:
    </p>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-lg-4"
  >
    <a
      href="https://charat.me/"
    >
      <AvatarImage />
    </a>
  </div>
</div>
`;
